diff --git a/node_modules/react-native-date-picker/ios/RNDatePicker.mm b/node_modules/react-native-date-picker/ios/RNDatePicker.mm
index 8fe3df4..5a5e46c 100644
--- a/node_modules/react-native-date-picker/ios/RNDatePicker.mm
+++ b/node_modules/react-native-date-picker/ios/RNDatePicker.mm
@@ -1,4 +1,5 @@
 #import "RNDatePicker.h"
+#include <string>
 
 #ifdef RCT_NEW_ARCH_ENABLED
 #import "RCTConvert.h"
